models/stg_hubspot__company.sql

```diff
@@ -63,7 +63,10 @@ with base as (
     from macro
 
 {% endif %}
+
+), joined as (
+    {{ add_property_labels('hubspot__company_pass_through_columns', cte_name='fields') }}
```

Comment:
This is only for fields that are explicitly declared in the passthrough variable, right? What if a customer is using the `hubspot__pass_through_all_columns: true` config and having all the `property_hs` columns included without needing to explicitly update the `hubspot__company_pass_through_columns` variable? Or is that not really a need here?

I could also see that causing the model to be VERY computation expensive since joins and subqueries would be created for every single `property_hs` field.

Reply:
I also talked about this in response to your comment on `macros/add_property_labels`. But I purposefully opted not to allow this. Also see scoping comments in the ticket, but I'll paste the section that also influenced me:

> Other Considerations
This becomes a lot more complicated for people who have hubspot__pass_through_all_columns set to true. The customer who opened the ticket specifically asked for this to be incorporated into the individual hubspot__[table_name]_pass_through_columns vars, so perhaps we should just address this first and then address the other scenario if a customer opens a ticket asking for it. Open to opinions!